SCAutolib/enums.py

```diff
@@ -1,15 +1,17 @@
 from enum import Enum, auto
 
 
-class OSVersion(Enum):
+class OSVersion(int, Enum):
```

Comment:
1. if you are not interested in the actual value of the enum, you can use `auto()` for the value
2. Check IntEnum, or StrEnum (for version >= 3.11)

Reply:
Good idea but as said we want compatibility at least until Python 3.6 so this is not really an option.